src/main/java/at/qcademy/tasks/oop/BuchundAutor/AppBuch.java

```diff
@@ -0,0 +1,34 @@
+package OOP.BuchundAutor;
+
+
+import OOP.Person.Person;
+
+public class AppBuch {
+
+    public static void main(String[] args) {
+        Bibliothek bibliothek = new Bibliothek();
+        Buch buch1 = new Buch("Titel1", "ISBN1", new Autor("Max Mustermann", "Muster", 20));
+        Buch buch2 = new Buch("Titel2", "ISBN2", new Autor("Maria Musterfrau", "Musterfrau", 25));
+        Buch buch3 = new Buch("Titel3", "ISBN3", new Autor("Ekrem Gedik", "ako", 50));
+
+
+        Autor autor = new Autor("Ako R", "Ako", 50);
```

Comment:
Why do you need this autor here?

Reply:
for creating a writer example

Reply:
Good, but you don't use it anywhere except printing it out.